pkgs/development/hare-third-party/hare-ssh/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, hare, harec, hareThirdParty, fetchFromSourcehut }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hare-ssh";
+  version = "unstable-2023-11-16";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = "hare-ssh";
+    rev = "c6a39e37ba4a42721594e0a907fe016f8e2198a8";
+    hash = "sha256-I43TLPoImBsvkgV3hDy9dw0pXVt4ezINnxFtEV9P2/M=";
+  };
+
+  nativeBuildInputs = [ hare ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    export HARECACHE="$NIX_BUILD_TOP/.harecache"
+    export BINOUT="$NIX_BUILD_TOP/.bin"
+
+    makeFlagsArray+=(
+      PREFIX="${builtins.placeholder "out"}"
+    )
+
+    runHook postConfigure
+  '';
+
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://git.sr.ht/~sircmpwn/hare-ssh/";
+    description = "SSH client & server protocol implementation for Hare";
+    license = with licenses; [ mpl20 ];
+    maintainers = with maintainers; [ patwid ];
+
+    inherit (harec.meta) platforms badPlatforms;
```

Remove the use of `harec`

```suggestion
    inherit (hare.meta) platforms badPlatforms;
```

---

Applied